tests/tree_inference/test_mcmc.py

```diff
@@ -191,3 +191,6 @@ def test_swap_subtrees_move():
 
     assert jnp.array_equal(new_tree.tree_topology, new_tree_corr.tree_topology)
     assert jnp.array_equal(new_tree.labels, new_tree_corr.labels)
+
+
+# TODO: test MCMC Kernel - any suggestions?
```

Comment:
@pawel-czyz Given the `log_prob` is not implemented, I could code some tests for the yet-to-be-tested `mcmc_kernel`. Any ideas?

Reply:
I think we actually have it – isn't log-prob just the log-likelihood function in our case? We have (a) fixed rates, (b) uniform prior on the trees, (c) uniform attachments, so the difference between these two is the log of the normalization constant.

Reply:
Yes, indeed, not many additional workings in the `mcmc_kernel`. I will skip a test here then. Thanks for double-checking this with me.